base/src/MotionVectorExtractor.cpp

```diff
@@ -244,8 +264,15 @@ void DetailOpenH264::getMotionVectors(frame_container& frames, frame_sp& outFram
 	int32_t mMotionVectorSize = mWidth * mHeight * 8;
 	int16_t* mMotionVectorData = nullptr;
 	memset(&pDstInfo, 0, sizeof(SBufferInfo));
-	outFrame = makeFrameWithPinId(mMotionVectorSize, motionVectorPinId);
-	mMotionVectorData = static_cast<int16_t*>(outFrame->data());
+	if (sendOverlayFrame) 
```

Comment:
formatting, in vs code use Ctrl+K, F to auto format

Reply:
Done